README.md

```diff
@@ -212,9 +331,9 @@ function Counter({ multiplier }) {
 
 In the above example, any change to `multiplier` prop will show up in the `observableProps` observable object, and be picked up by the `store`.
 
-Warning: _the return value of `useAsObservableSource` should never be deconstructed! So, don't write: `const {multiplier} = useAsObservableSource({ multiplier })`!_
+Warning: \_the return value of `useAsObservableSource` should never be deconstructed! So, don't write: `const {multiplier} = useAsObservableSource({ multiplier })`!\_useObservable
 
-The value passed to `useAsObservableSource` should always be an object, and is made only shallowly observable.
+The value passeduseObservableuld always be an object, and is made only shallowly observable.
```

Comment:
Wait what it `passeduseObservableuld`? :D

Reply:
Me fighting with the scroll wheel :P :computer_mouse: